provider/google.go

```diff
@@ -159,6 +159,10 @@ func (p *GoogleProvider) Zones() (map[string]*dns.ManagedZone, error) {
 		return nil, err
 	}
 
+	for _, zone := range zones {
+		log.Debugf("Considering zone: %s (domain: %s)", zone.Name, zone.DnsName)
+	}
```

Comment:
@linki @ideahitme thoughts on the more verbose behavior for this feature in [PR #388 - google.go:152](https://github.com/kubernetes-incubator/external-dns/pull/388/files#diff-ffadcf5d29c57da1fa268c4821471734R152) ?